libs/design/badge/src/badge.module.ts

```diff
@@ -0,0 +1,31 @@
+import { CommonModule } from '@angular/common';
```

Comment:
Since this is a completely new component, the module is unnecessary to create and immediately deprecate. It can be removed.